fbmuck/src/version.tpl

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 1991-2009 by Fuzzball Software
```

Comment:
Just copied this from the existing version.sh/c file. I can update this in a later pull request, or someone else on the project can if they so choose.